cmd/podman/README.md

```diff
@@ -109,3 +109,11 @@ The complete set can be found in the `validate` package, here are some examples:
     created := validate.ChoiceValue(&opts.Sort, "command", "created", "id", "image", "names", "runningfor", "size", "status")
     flags.Var(created, "sort", "Sort output by: "+created.Choices())
     ```
+
+## Adding CLI flags
+
+When adding adding anew cli option that accepts a string array then there are two options to choose from: `StringSlice()` and `StringArray()`.
```

Comment:
- Missing space: "a new", not anew
- suggestion, change "a string array then there are" -> "a string array, there are"

Apologies in advance for possible email bombardment. This is a big PR, and I've been bitten before by github trashing review comments when someone repushes, so I am going to submit feedback one by one. There is no need to repush for every comment. I will give an "all done" signal when I've finished my first pass.